test/__snapshots__/minify-option.test.js.snap

```diff
@@ -1,8 +1,125 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`when applied with \`minify\` option matches snapshot for \`terser\` minifier and \`parallel\` is \`true\`: errors 1`] = `Array []`;
+exports[`when applied with \`minify\` option matches snapshot for \`terser\` minifier and \`parallel\` is \`true\`: errors 1`] = `
+Array [
+  "Error: main.js from Terser
+SyntaxError: Unexpected string",
```

Comment:
@filipesilva Problem here :confused:

Reply:
I can't reproduce :/

Tried running it multiple times as well to see if it was a race condition or something.

If I can help let me know. Otherwise I worry that I'm just bothering you while you work.

Reply:
@filipesilva thanks for helping, do not worry, i will ping you if can't solve problem (but it is unlikely) :smile: